tests/views/test_message.py

```diff
@@ -192,12 +192,13 @@ def test_technical_inbox_threads_list(self, mock_request, mock_get_jwt):
         of the messages.  Once we do, we can know what the requirements are (i.e., will messages of certain categories
         have survey, business and collection exercise information?)
         """
+        thread_id = "ff4537df-2097-4a73-a530-e98dba7bf28f"
```

Comment:
Strictly speaking this isn't quite right, although this will give line coverage, the fact that no additional assert has been added tells me that something is not right, whether it be another assert is needed or the code isn't working as expected. We should also have clear tests that focus on one functional part, there should be a test just for a thread_id being passed to the page and the result, and it should be named appropriately